spec/models/manageiq/providers/azure/cloud_manager_spec.rb

```diff
@@ -358,11 +358,11 @@ def create_factory_ems(name, region)
     end
   end
 
-  context 'catalog types' do
+  describe "#catalog types" do
```

Comment:
@djberg96 I don't do describe vs context based on location (inner vs outer).  I tend to recommend more like what @agrare said:

- Use `describe` to describe the method under test, trying to use the actual method name if possible (e.g. `".the_method"` for class methods, `"#the_method"` for instance methods and appending `" (private)"` if the method under test is private).
- Use `context` to describe a particular environment in which the tests runs.  (e.g. `"when there are credentials defined"`), and then having a `before` that sets up specific credentials
- I tend to avoid creating extraneous `describe`s and `context`s unless I have more than one test.  That is, if a single `it` is good enough, I just do that...

   ```ruby
   # I don't really like:
   describe ".the_method" do
     it "works" do
       described_class.the_method
     end
   end

   # I prefer:
   it ".the_method" do
     described_class.the_method
   end
   ```